# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../psutils/destdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- psutils/Makefile.unix.orig	1997-03-11 23:52:59.000000000 +0100
+++ psutils/Makefile.unix	2007-08-17 16:50:30.000000000 +0200
@@ -222,27 +222,27 @@
 	-mkdir $(BINDIR)
 	@for i in $(BIN); do \
 		echo Installing $$i; \
-		$(INSTALL) $$i $(BINDIR); \
+		$(INSTALL) $$i $(DESTDIR)$(BINDIR); \
 	done
 
 install.script: $(PERLSCRIPTS) $(SHELLSCRIPTS)
 	-mkdir $(SCRIPTDIR)
 	@for i in $(PERLSCRIPTS) $(SHELLSCRIPTS); do \
 		echo Installing $$i; \
-		$(INSTALL) $$i $(SCRIPTDIR); \
+		$(INSTALL) $$i $(DESTDIR)$(SCRIPTDIR); \
 	done
 
 install.include: $(INCLUDES)
 	-mkdir $(INCLUDEDIR)
 	@for i in $(INCLUDES); do \
 		echo Installing $$i; \
-		$(INSTALLMAN) $$i $(INCLUDEDIR); \
+		$(INSTALLMAN) $$i $(DESTDIR)$(INCLUDEDIR); \
 	done
 
 install.man: $(MANPAGES)
 	-mkdir $(MANDIR)
 	@for i in $(MANPAGES); do \
 		echo Installing manual page for $$i; \
-		$(INSTALLMAN) $$i $(MANDIR)/$$i; \
+		$(INSTALLMAN) $$i $(DESTDIR)$(MANDIR)/$$i; \
 	done
 
